web/profiles/joinup/src/Plugin/Block/AddContentBlock.php

```diff
@@ -73,9 +84,10 @@ public function build() {
     // caching ourselves.
     /** @var \Drupal\Core\Plugin\Context\Context[] $collection_contexts */
     $collection_contexts = $this->collectionContext->getRuntimeContexts(['collection']);
-    if ($collection_contexts['collection']->hasContextValue()) {
+    if ($collection_contexts && $collection_contexts['collection']->hasContextValue()) {
       $page_url = Url::fromRoute('custom_page.collection_custom_page.add', [
-        'rdf_entity' => $collection_contexts['collection']->getContextValue()->id(),
+        'rdf_entity' => $collection_contexts['collection']->getContextValue()
+          ->id(),
```

Comment:
This is maybe one thing about the codesniffer.
The issue is that during a git push there is a check. But if there are problems in the coding standards and the user fixes them, the phpcs goes green regardless of whether the changes were committed.